content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -3,3 +3,41 @@ title: "Workload Resources"
 weight: 20
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+You use Kubernetes API to create {{< glossary_tooltip text="object" term_id="object" >}}
+(HTTP resources) that represent these higher level abstractions, and then let the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manage
+Pods on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) and [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)
+(replacing the legacy resource
+{{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}).
+`Deployment` is a good fit for managing a stateless application workload on your cluster,
+where any {{< glossary_tooltip term_id="Pod" >}} in the `Deployment` is interchangeable
+and can be replaced if needed.
+
+A [`StatefulSet`](/docs/concepts/workloads/controllers/statefulset/) lets you
+run one or more related Pods that do track state somehow. For example, if your workload
```

Comment:
nit: "related" may give a connotation that they are not the same Pod. I made the following suggestions since the pods of a StatefulSet uses replicas and identical "container spec" (https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) for its pods
```suggestion
run one or more replicated Pods that do track state somehow. For example, if your workload
```

Reply:
Hmm, replicated could also mislead readers into thinking they get a feature that Kubernetes doesn't provide.

Reply:
@reylejano any thoughts on how to avoid that confusion (people thinking that Kubernetes provides a facility for replicating persistent storage)? 

I noticed this was only a nit.